sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/ErrorResponse.cs

```diff
@@ -57,34 +42,10 @@ public ErrorResponse(string code = default(string), string message = default(str
         partial void CustomInit();
 
         /// <summary>
-        /// Gets the error code.
+        /// Gets or sets the error details.
         /// </summary>
-        [JsonProperty(PropertyName = "code")]
-        public string Code { get; private set; }
-
-        /// <summary>
-        /// Gets the error message.
-        /// </summary>
-        [JsonProperty(PropertyName = "message")]
-        public string Message { get; private set; }
-
-        /// <summary>
-        /// Gets the error target.
-        /// </summary>
-        [JsonProperty(PropertyName = "target")]
-        public string Target { get; private set; }
-
-        /// <summary>
-        /// Gets the error details.
-        /// </summary>
-        [JsonProperty(PropertyName = "details")]
-        public IList<ErrorResponse> Details { get; private set; }
-
-        /// <summary>
-        /// Gets the error additional info.
-        /// </summary>
-        [JsonProperty(PropertyName = "additionalInfo")]
-        public IList<ErrorAdditionalInfo> AdditionalInfo { get; private set; }
```

Comment:
this is a breaking change, likely caused by the directory you are generating into.